benchmarks/basic.js

```diff
@@ -9,8 +9,15 @@ var fs = require('fs')
 var dest = fs.createWriteStream('/dev/null')
 var loglevel = require('./loglevelMock')(dest)
 var plog = pino(dest)
+
+var tty = require('tty')
+var extremeDest = new tty.WriteStream(1)
+extremeDest.write = function (s) {
+  // simulate a real write so that the benchmark is not artificially low
+  dest.write(s)
+}
 delete require.cache[require.resolve('../')]
-var plogExtreme = require('../')({extreme: true}, dest)
+var plogExtreme = require('../')({extreme: true}, extremeDest)
```

Comment:
Sure, we could do the following to get a stream with a direct file descriptor:

```js
var fd = fs.openSync('/tmp/extreme', 'w')
var extremeDest = fs.createWriteStream('/tmp/extreme', {fd: fd})
```

But just `fs.createWriteStream('/tmp/extreme')` (or to `/dev/null`) results in a stream without a direct file descriptor attached; the property is set to `null`. Thus we have to do *something* to get a blockable stream. I just opted for the TTY.